use-timescale/backup-restore.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Back up and recover your Tiger Cloud services
+excerpt: See how and when Tiger Cloud backs up your data, making sure you always have something to fall back on in case of disaster recovery
+products: [cloud]
+keywords: [backups, restore]
+tags: [recovery, failures]
+---
+
+# Back up and recover your $SERVICE_SHORTs
+
+$CLOUD_LONG automatically handles backup for your $SERVICE_LONGs using the `pgBackRest` tool. You don't need to perform backups manually. What's more, with [cross-region backup][cross-region], you are protected even if an entire AWS region goes down.
+
+$CLOUD_LONG automatically creates one full backup every week, and
+incremental backups every day in the same region as your $SERVICE_SHORT. Additionally, all WAL ([Write-Ahead Log][wal])
+files are retained back to the oldest full backup. This means that you always
+have a full backup available for the current and previous week: 
+
+![Backup in Tiger Cloud](https://assets.timescale.com/docs/images/database-backup-recovery.png)
+
+In the event of a storage failure, a $SERVICE_SHORT automatically recovers from backup
+to the point of failure. If the whole availability zone goes down, your $SERVICE_LONGs are recovered in a different zone. In the event of a user error, where a point-in-time recovery needs to be done, you can [create a point-in-time recovery fork][create-fork].
+
+## Enable cross-region backup 
+
+<Availability products={['cloud']} price_plans={['enterprise']} />
+
+For added reliability, you can enable cross-region backup. In this case, you have two identical backups of your $SERVICE_SHORT at any time, but one of them is in a different AWS region.
```

Comment:
```suggestion
For added reliability, enable cross-region backup. This protects your data when an entire AWS region goes down. Cross-region backups are identical replicas of your $SERVICE_SHORT, but one of them is in a different AWS region. You can have one cross-region backup for a $SERVICE_SHORT.
```

Reply:
Cross-region backups are identical replicas of your $SERVICE_SHORT, but one of them is in a different AWS region --> this reads like you have multiple cross-region backups per service, which is not the case. Also I wanted to clarify that you don't create cross-region backup once and it stays the same - it gets updated daily and weekly just like the regular backup, so at any point in time you have two identical ones - cross and same-region.